src/server/coprocessor/executor/mod.rs

```diff
@@ -14,6 +14,30 @@
 use util::codec::table::RowColsDict;
 use server::coprocessor::Result;
 
+#[allow(dead_code)] //TODO:remove it
+pub struct Row {
+    handler: i64,
+    data: RowColsDict,
+}
+
+#[allow(dead_code)] //TODO:remove it
+impl Row {
+    pub fn new(handler: i64, data: RowColsDict) -> Row {
+        Row {
+            handler: handler,
+            data: data,
+        }
+    }
+
+    pub fn get_mut_data(&mut self) -> (&mut RowColsDict) {
```

Comment:
The `handle`  once created, it cannot be modified. It seems `RowColsDict` is enough @siddontang